src/ui/component/transactionList/view.jsx

```diff
@@ -1,156 +1,110 @@
+                <option key={tt} value={tt}>
+                  {__(`${capitalize(tt)}`)}
+                </option>
+              ))}
+            </FormField>
+          </div>
         </header>
-        {!slim && !!transactions.length && (
-          <header className="table__header">
-            <div className="card__actions--between">
-              <FileExporter
-                data={transactionList}
-                label={__('Export')}
-                title={__('Export Transactions')}
-                filters={['nout']}
-                defaultPath={__('lbry-transactions-history')}
-              />
-
-              <FormField
-                type="select"
-                name="file-sort"
-                value={filterSetting || TRANSACTIONS.ALL}
-                onChange={this.handleFilterChanged}
-                label={__('Show')}
-                postfix={
-                  <Button
-                    button="link"
-                    icon={icons.HELP}
-                    href="https://lbry.com/faq/transaction-types"
-                    title={__('Help')}
-                  />
-                }
-              >
-                {transactionTypes.map(tt => (
-                  <option key={tt} value={tt}>
-                    {__(`${this.capitalize(tt)}`)}
-                  </option>
-                ))}
-              </FormField>
-            </div>
-          </header>
-        )}
+      )}
 
-        {!loading && !transactionList.length && (
-          <h2 className="main--empty empty">{emptyMessage || __('No transactions.')}</h2>
-        )}
+      {!loading && !transactions.length && (
+        <h2 className="main--empty empty">{emptyMessage || __('No transactions.')}</h2>
+      )}
 
-        {!!transactionList.length && (
-          <React.Fragment>
-            <table className="table table--transactions">
-              <thead>
-                <tr>
-                  <th>{__('Amount')}</th>
-                  <th>{__('Type')} </th>
-                  <th>{__('Details')} </th>
-                  <th>{__('Transaction')}</th>
-                  <th>{__('Date')}</th>
-                </tr>
-              </thead>
-              <tbody>
-                {transactionList.map(t => (
-                  <TransactionListItem
-                    key={`${t.txid}:${t.nout}`}
-                    transaction={t}
-                    reward={rewards && rewards[t.txid]}
-                    isRevokeable={this.isRevokeable(t.txid, t.nout)}
-                    revokeClaim={this.revokeClaim}
-                  />
-                ))}
-              </tbody>
-            </table>
-          </React.Fragment>
-        )}
-      </React.Fragment>
-    );
-  }
+      {!!transactions && !!transactions.length && <TransactionListTable transactionList={transactions} />}
+      {!slim && !!transactionCount && (
+        <Paginate
+          onPageChange={page => history.replace(`#/$/transactions?page=${Number(page)}`)}
```

Comment:
You should use the `PAGES` constants here. And you don't need the `#`. The router adds that automatically for the platform you're on. The hash isn't included on lbry.tv